### DISCUSSION TITLE
how to import hooks files into md?

### DISCUSSION THREAD

**Discussion by cunzaizhuyi** [vuepress/core#390] [Q&A] [answered]
here is how to import hooks files into one .md document:

![image](https://user-images.githubusercontent.com/20167257/129658960-6a23fd61-905c-4f15-a62a-26de22cd4a2c.png)

here is the hooks file test.js:
![image](https://user-images.githubusercontent.com/20167257/129659095-66ac0b9d-e502-47e7-9055-1f5b137aa0f9.png)

here is the error in terminal:
![image](https://user-images.githubusercontent.com/20167257/129659169-783699ae-e9ca-4f91-adac-b8b877482d2a.png)

so, how to import hooks files into .md?

**Comment by Mister-Hope** [comment, marked as answer]
All markdown pages wil be compiled to vue sfc under `pages` in temp folder, and that temp folder obviously doesn't contain that file.

You need to create an alisa for that file and import that file using alias.
